Production/govuk_ios/Helpers/StringValidator/ValidatorProvider.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+protocol ValidatorProvider {
+    func validate(input: String?) -> Bool
+}
+
+struct MaximumLengthValidator: ValidatorProvider {
+    private let requiredStringLength: Int
+
+    init(requiredStringLength: Int) {
```

Comment:
`requiredStringLength` is potentially a little misleading here as if you had a required string length of 5, you would expect 4 and 6 to fail, although here, 4 would be considered valid.
Suggest just calling this var "length" or "value"